islamapp/model/data_models/user_followers.py

```diff
@@ -0,0 +1,26 @@
+from mongoengine import fields, Document
+
+class UserFollowers(Document):
+    
+    scraped_user_id= fields.IntField(primary_key=True,required=True)
+    scraped_user = fields.StringField(required=True)
+    create_at = fields.DateTimeField(required=True)
+    followers_count = fields.IntField(required=True)
+    followers_list = fields.ListField(required=True)
+    followers_change_count = fields.IntField(required=True ,default=0)
```

Comment:
What does this stands for (just curious)

Reply:
I think maybe follower change will be useful in the future. Like showing the changes in user's relation or user followers trends. It will compare to the last scrape data.

Reply:
It strikes me that maybe we still need a auto incremental primary key if we want to store scraped data every time and delete update_at?